### ISSUE TITLE
Option to update your profile details 

### ISSUE THREAD

**Issue by rokublac** [scrimba/community#411]
It would be nice to have an option to change your profile details as currently it seems to be set once you create an account or log in via GitHub. If GitHub users on Scrimba decide to update their GitHub profile details, the changes is not reflected on their Scrimba profile page.

For example, if you decide to change your GitHub handle, Scrimba will still display and link out your previous handle, which most likely now directs to a `404`. This problem also extends to your profile name where if you log in the first time via GitHub and your GitHub does not have a profile name, your Scrimba profile name will be hard set as _undefined_.

**Comment by perborgen** [comment]
Yes, totally agree that we should have a way to update this. The way we've solved this until now is by emailing me, and then I fix it manually. That works for now, but I'll look into creating a better solution.

**Comment by scanf** [comment]
Related-to: scrimba/community#440
